snuba/web/rpc/v1/endpoint_trace_item_table.py

```diff
@@ -235,14 +237,19 @@ def _execute(self, in_msg: TraceItemTableRequest) -> TraceItemTableResponse:
         in_msg = _apply_labels_to_columns(in_msg)
         _validate_select_and_groupby(in_msg)
         _validate_order_by(in_msg)
-        snuba_request = _build_snuba_request(in_msg)
+        request_id = in_msg.meta.request_id or str(uuid.uuid4())
```

Comment:
I would just enrich the object itself rather than adding a special function parameter